### PR TITLE
added two visualizations + all graphs

### PR DESCRIPTION
**Pull request by Carol217**
Graphs will most likely be overwritten with gutenberg data, but all existing sample data is now published. Graph-style markdown has been updated with useful information. at a later date, a new function to sort inst_dist might be added.

### CODE REVIEW

File: gender_novels/analysis/gender_pronoun_freq_analysis.py [hunk 1]
```diff
@@ -372,6 +386,156 @@ def sort_every_year(frequency_dict):
...
+                     "Author Gender")
+    female_date_sub_v_ob = get_mean(freq_by_date(sub_v_ob[1]))
+    male_date_sub_v_ob = get_mean(freq_by_date(sub_v_ob[0]))
+    bar_sub_obj_freq(female_date_sub_v_ob, male_date_sub_v_ob, "obj_sub_by_year", "Years")
+
+    female_loc_sub_v_ob = get_mean(freq_by_location(sub_v_ob[1]))
+    male_loc_sub_v_ob = get_mean(freq_by_location(sub_v_ob[0]))
+    bar_sub_obj_freq(female_loc_sub_v_ob, male_loc_sub_v_ob, "obk_sub_by_location", "Location")
+
+    # male_tot = dict_to_list(sub_v_ob[0])
```
Comment (ltagliaferri) [inline gender_novels/analysis/gender_pronoun_freq_analysis.py]: Can you clean up these commented out print statements and then update the PR please?

Comment (ltagliaferri) [inline gender_novels/analysis/gender_pronoun_freq_analysis.py]: Can you clean up these commented out print statements and then update the PR please?

File: gender_novels/analysis/gender_pronoun_freq_analysis.py [hunk 2]
```diff
@@ -372,6 +386,156 @@ def sort_every_year(frequency_dict):
...
+                palette=my_pal).set_title("Relative Frequency of Female Pronouns to Total Pronouns")
+    plt.xticks(rotation=90)
+    # plt.show()
+
+    filepng = "visualizations/" + title + ".png"
+    filepdf = "visualizations/" + title + ".pdf"
+    plt.savefig(filepng, bbox_inches='tight')
+    plt.savefig(filepdf, bbox_inches='tight')
+
+    return
```
Comment (elenaboal) [inline gender_novels/analysis/gender_pronoun_freq_analysis.py]: Hi, this isn't pertinent, but if you aren't returning anything, this statement is unnecessary

File: gender_novels/analysis/gender_pronoun_freq_analysis.py [hunk 3]
```diff
@@ -372,6 +386,156 @@ def sort_every_year(frequency_dict):
...
+    :param x: value of x axis
+    :return:
+    """
+
+    fig, ax = plt.subplots()
+    plt.ylim(0, 1)
+
+    key = []
+
+    for k, v in she_freq_dict.items():
```
Comment (elenaboal) [inline gender_novels/analysis/gender_pronoun_freq_analysis.py]: Maybe try to use more descriptive variable names to make it more readable and help people unfamiliar with what keys and values are associated with she_freq_dict

File: gender_novels/analysis/gender_pronoun_freq_analysis.py (hunk 1 above)
Comment (Carol217) [reply gender_novels/analysis/gender_pronoun_freq_analysis.py]: yes, of course

File: gender_novels/analysis/gender_pronoun_freq_analysis.py (hunk 3 above)
Comment (Carol217) [reply gender_novels/analysis/gender_pronoun_freq_analysis.py]: the issue with this is that the key values vary. sometimes it'll be the author gender, other times the date. this function is intended to be run with a data set sorted by the 'freq_by_...' functions. i'll include that in the documentation

**Pull request closed without merging**